SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -66,6 +68,10 @@ class FLocalDeploymentManager
 	void StartUpWorkerConfigDirectoryWatcher();
 	void OnWorkerConfigDirectoryChanged(const TArray<FFileChangeData>& FileChanges);
 
+	bool FinishLocalDeployment(FString LaunchConfig, FString RuntimeVersion, FString LaunchArgs, FString SnapshotName, FString RuntimeIPToExpose);
```

Comment:
This could just return void now I reckon.

Reply:
I think I will actually use its return in TryStartLocalDeployment instead.